_src/css/_body.scss

```diff
@@ -16,7 +16,29 @@
     margin: 25px;
     width: 50%;
     display: flex;
+  }
+
+  &__frame {
+    display: flex;
+    align-items: center;
+    padding-left:25 px;
+  }
+  
```

Comment:
On this empty line you are going to create a rule using the pseudo-element `::after`. In order to have the output correctly working we use two very important rules: 
`&__frame::after {
    content: "";
    position: absolute;
}`

Once these rules are applied, you can start tweaking.

You should also note that the element `&__frame` will need a `position: relative` rule to have the `::after: position: absolute` working as expected